Assets/Scripts/Questing/Party/NPC/AttackScripts/Scripts/NPC_ArcherAttack.cs

```diff
@@ -71,5 +71,6 @@ public override void EndAttack()
     public void AttackTarget()
     {
         aTarget.currentTarget.GetComponent<HitPointManager>().EffectHealth(attackDamge);
+        thisAdventurer.GetComponent<Animator>().SetTrigger("Attack");
     }
```

Comment:
I guess this also applies to treasure? Because the archer can pick up the treasure from afar.

Reply:
yeah, they just attack the loot